Pspcl/Pspcl.Services/Models/AddUserModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Pspcl.Core.Domain;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Pspcl.Services.Models
+{
+    public class AddUserModel : IdentityUser<int>
```

Comment:
Who told you to inherit it from Identity class ? Tum pehle ye pdho model and entities mein kya difference hota hai